app/services/feature_flag.rb

```diff
@@ -18,6 +18,7 @@ def feature
     [:dfe_sign_in_fallback, 'Use this when DfE Sign-in is down', 'Tijmen Brommet'],
     [:force_ok_computer_to_fail, 'OK Computer implements a health check endpoint, this flag forces it to fail for testing purposes', 'Michael Nacos'],
     [:pilot_open, 'Enables the Apply for Teacher Training service', 'Tijmen Brommet'],
+    [:preview_2021_recruitment_cycle, 'Sync and serve courses for the 2021 recruitment cycle. DO NOT ENABLE IN PRODUCTION.', 'Duncan Brown'],
```

Comment:
How about `:use_2021_recruitment_cycle` instead? We could actually use this feature flag for real to migrate production environments to the new cycle.

Reply:
Yes, preview's not much good! `use_` is not bad... but really you don't "use" a recruitment cycle, you're sort of in it. Maybe `switch_to_`? `enter_`?

Reply:
`:switch_to_2021_recruitment_cycle` is great